gnd/src/androidTest/java/com/google/android/gnd/CustomTestRunner.java

```diff
@@ -0,0 +1,14 @@
+package com.google.android.gnd;
+
+import android.app.Application;
+import android.content.Context;
+import androidx.test.runner.AndroidJUnitRunner;
+
+public class CustomTestRunner extends AndroidJUnitRunner {
```

Comment:
Can we mark the custom module, injector, and test runner somehow (name/package/path) to make it clear that they exist to allow integration testing? It seems it would be easy to conflate integration and unit testing; for unit testing, for example, we probably want to instantiate and supply the dependencies ourselves to constructors, so the custom test graph wouldn't apply.

Reply:
They're all in the `androidTest` folder which is designed for instrumented tests (i.e. integration and e2e tests) - is that sufficient?